pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -104,10 +112,14 @@ func NewFit(plArgs runtime.Object, _ framework.Handle) (framework.Plugin, error)
 		return nil, err
 	}
 
-	return &Fit{
+	fit := &Fit{
 		ignoredResources:      sets.NewString(args.IgnoredResources...),
 		ignoredResourceGroups: sets.NewString(args.IgnoredResourceGroups...),
```

Comment:
should we have the beta features enabled by default? It might save some surprises in tests or other unforeseen users

Reply:
this is going to be hard to track though, and so will likely cause inconsistencies.